homeassistant/package_constraints.txt

```diff
@@ -117,7 +117,7 @@ backoff<2.0
 
 # Breaking change in version
 # https://github.com/samuelcolvin/pydantic/issues/4092
-pydantic!=1.9.1
+pydantic=1.10.0a1
```

Comment:
(I've corrected the constraint)

It looks like `SystemBridgeCoordinatorData` is the only pydantic model in this repo.

If you think there's no value in running these tests, feel free to close this.

If dependants of this repo use pydantic, I'll try to test them directly, but if dependencies use pydantic, running these tests would be good if possible.

Reply:
> `SystemBridgeCoordinatorData`

Hmm, that seems incorrect / a bug, as that integration doesn't specify having a dependency on pedantic.

> If you think there's no value in running these tests, feel free to close this.

It was not a question about value :) I was a question about this not doing what you probably expected it would do; which is running our test suite against `pydantic 1.10.0a1`.

Reply:
Okay, in that case close this.

By the way, it's probably worth excluding `pydantic>2` since pydantic V2 will break a lot of things.

Reply:
That is up to the dependencies to handle correctly. We only add constraints when needed.